### DISCUSSION TITLE
Setup with one dagster webserver and 2 code locations served by different grpc server

### DISCUSSION THREAD

**Discussion by anotine10** [dagster-io/dagster#20533] [Q&A]
Hi,
I am looking at using dagster in our own cloud (ECS) with 2 different teams working on their Assets.

Is there some updated doc that shows how to setup the following:
- Dagster instance code deployed in ECS
- 2 Code locations deployed separately in ECS and serving through gRPC server.

How the CI/CD pipeline would be in order to merge changes code changes in one of the code definitions deployment? 
How the models are reloaded from the Dagster webserver container? 
If i add a third code location, is there  no other way to to deploy again the dagster instance code because of this workspace.yaml being changed? 

Thanks for you help

**Comment by garethbrickman** [comment]
In case you haven't encountered these resources yet: 

[There is example code and instructions for an ECS deployment in this repo](https://github.com/dagster-io/dagster/tree/1.6.11/examples/deploy_ecs#readme)

[These docs contain information about running your own gRPC server for your code locations](https://docs.dagster.io/concepts/code-locations/workspace-files#running-your-own-grpc-server)

**Reply by anotine10** [reply]
Thanks @garethbrickman , actually i had already gone through that documents.
The first link is using a docker ecs integration which looks deprecated from docker (https://docs.docker.com/cloud/), i was hopping a more updated doc and solution.

Regarding the different gRPC server. If i understand well, if i have 2 different code locations, i will have to deploy 2 different grpc in 2 different fargate/ec2 instance, right?

**Comment by bharris6avera** [comment]
@anotine10  Did you happen to find a solution? I am running a similar use case and I was wondering how you resolved this issue.
